travis.sh

```diff
@@ -25,7 +25,7 @@ elif [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ -n "${GITHUB_TOKEN-}" ]; then
 	# http://docs.travis-ci.com/user/pull-requests/#Security-Restrictions-when-testing-Pull-Requests
 	# That's why the analysis does not need to be executed if the variable GITHUB_TOKEN is not defined.
 	echo "Build and analyze internal pull request..."
-	mvn clean verify sonar:sonar -B -e -V \
+	mvn -P sonar clean verifygit  -e -V \
```

Comment:
@npetzall could you check this line, I can't find Maven goal `verifygit `?

Reply:
That git shouldn't be there.

Reply:
I'm blaming my son he was sitting in my lap at the time.

Reply:
haha good :smile: I know how it is coding with child on the knees